mobile_rewards/src/server.rs

```diff
@@ -441,7 +434,7 @@ async fn reward_period(client: &mut follower::Client<Channel>) -> Result<Range<u
 }
 
 pub fn get_time_range(last_reward_end_time: i64) -> (DateTime<Utc>, DateTime<Utc>) {
-    let after_utc = datetime_from_epoch(last_reward_end_time);
+    let after_utc = Utc.timestamp(last_reward_end_time, 0);
```

Comment:
why not use TimestampDecode for this and either impl for i64 or else just cast `as u64`?

Reply:
Because this was the _only_ place that would need that.. and casting as u64 looses the negative timestamp ability which by the contract of get_time_range is allowed

Reply:
that makes sense; i just figured it was because postgres expects them to be i64s